experiment/autobumper/bumper/bumper.go

```diff
@@ -38,26 +42,197 @@ const (
 	testImagePrefix = "gcr.io/k8s-testimages/"
 	prowRepo        = "https://github.com/kubernetes/test-infra"
 	testImageRepo   = prowRepo
+
+	latestVersion          = "latest"
```

Comment:
I feel these could be enums

Reply:
I'm not sure if enums could help here, since we need to allow users to configure it via a flag, so I left this unchanged.
If you have a better approach by using enums, please let me know.

Reply:
ack